appengine_flexible/memcache/memcache.go

```diff
@@ -70,3 +71,4 @@ func handle(w http.ResponseWriter, r *http.Request) {
 
 	fmt.Fprintf(w, "Count: %d", count)
 }
+// [END gae_flex_redislabs_memcache]
```

Comment:
Same as above, need a newline before `END` to make gofmt happy. Sorry about this.

Reply:
Thanks for finding all the gofmt errors. I will know next time to run it before I submit a PR!